### ISSUE TITLE
User isn't able to add white spaces in their names

### ISSUE THREAD

**Issue by abhi-status** [status-im/feature-specs#82]
# Bug Report

## Description

Currently, the user cannot add white spaces to their display names in the onboarding screen.

From John -
>  If allowing spaces in user names doesn't cause any issues, then we should allow spaces.  But we should check the @ mention auto complete behaviour if we make this change.
> e.g. spaces in names is 'nice to have', but not essential

#### Expected behavior
User must be allowed to continue with onboarding having white spaces in their names

#### Actual behavior
User is blocked by the validation
![image](https://user-images.githubusercontent.com/95302135/161762615-b79b6fd4-b102-4031-b519-1b3de026390d.png)

### Additional Information

- Status desktop version: Master : b54111f82ac0bdbecdace09358080f53891a5505
